client/sections.js

```diff
@@ -134,36 +134,6 @@ const sections = [
 		module: 'calypso/my-sites/posts',
 		group: 'sites',
 	},
-	{
-		name: 'settings-performance',
-		paths: [ '/settings/performance' ],
-		module: 'calypso/my-sites/site-settings/settings-performance',
-		group: 'sites',
-	},
-	{
-		name: 'settings-writing',
-		paths: [ '/settings/writing', '/settings/taxonomies', '/settings/podcasting' ],
-		module: 'calypso/my-sites/site-settings/settings-writing',
-		group: 'sites',
-	},
-	{
-		name: 'settings-discussion',
-		paths: [ '/settings/discussion' ],
-		module: 'calypso/my-sites/site-settings/settings-discussion',
-		group: 'sites',
-	},
-	{
-		name: 'settings-security',
-		paths: [ '/settings/security' ],
-		module: 'calypso/my-sites/site-settings/settings-security',
-		group: 'sites',
-	},
-	{
-		name: 'settings-jetpack',
-		paths: [ '/settings/jetpack' ],
-		module: 'calypso/my-sites/site-settings/settings-jetpack',
-		group: 'sites',
-	},
 	{
 		name: 'settings',
 		paths: [ '/settings' ],
```

Comment:
```suggestion
		paths: [ '/settings/general' ],
```

Reply:
@cpapazoglou This will not work as by changing that there is no `/settings` catch all route which should render the site picker ([https://wordpress.com/settings/](https://wordpress.com/settings/)). If you add `/settings` as well as `/settings/general` you will still get the exact same bug